sql/plan/create_view.go

```diff
@@ -161,3 +174,57 @@ func (cv *CreateView) WithDatabase(database sql.Database) (sql.Node, error) {
 	newCreate.database = database
 	return &newCreate, nil
 }
+
+// GetIsUpdatableFromCreateView returns whether the view is updatable or not.
+// https://dev.mysql.com/doc/refman/8.0/en/view-updatability.html
+func GetIsUpdatableFromCreateView(cv *CreateView) bool {
```

Comment:
Interesting! Thanks for leaving that doc link in the comments. I scanned that and it looks like the rules to determine updateability are fairly complex. Do we already support updating a view and writing the changes back to the source table? (I shoulda' just tested that, but didn't ;-) )